### PR TITLE
add pointpats recipe

### PR DESCRIPTION
**Pull request by slumnitz**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

@weikang9009 is a maintainer

### CODE REVIEW

File: recipes/pointpats/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  host:
+    - python
+    - pip
+  run:
+    - python >=3.4
+    - scipy>=0.11
+    - numpy>=1.3
+    - pandas
+    - matplotlib
+    - libpysal
```
Comment (weikang9009) [inline recipes/pointpats/meta.yaml]: Can you change this to `libpysal >=4.0.0`? thanks!

File: recipes/pointpats/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  sha256: 3aaceb5576b8a587e5b8b81877dfd0b87439a199e8a813af27086effbd9ed673
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
```
Comment (weikang9009) [inline recipes/pointpats/meta.yaml]: I think you have to set `python >=3`
image

File: recipes/pointpats/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3.4
+    - scipy>=0.11
```
Comment (chrisburr) [inline recipes/pointpats/meta.yaml]: The version constraints need to be formatted like this:
```suggestion
    - scipy >=0.11
```
i.e. one space after the name and nothing after the operator (`>`)

File: recipes/pointpats/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    - numpy>=1.3
```
Comment (chrisburr) [inline recipes/pointpats/meta.yaml]:
```suggestion
    - numpy >=1.3
```

File: recipes/pointpats/meta.yaml (hunk 3 above)
Comment (slumnitz) [reply recipes/pointpats/meta.yaml]: awesome, thanks!

**Pull request merged**